packages/app/pages/login.tsx

```diff
@@ -141,156 +141,143 @@ export default function Login(): ReactElement {
         <title>Login | HelpMe</title>
       </Head>
       <Container>
-        <Card className="sm:px-2 md:px-6">
-          <Row gutter={{ xs: 8, sm: 16, md: 24, lg: 32 }}>
-            <Col xs={{ span: 24 }} sm={{ span: 11 }}>
-              <h2 className="text-left">Hey, hello 👋</h2>
+        <Card className="mx-auto max-w-md sm:px-2 md:px-6">
+          <h2 className="text-left">Hey, hello 👋</h2>
```

Comment:
should we change the text here to something else? I feel the current thing is a little goofy. Though, I can't say I really know what to replace it with. Maybe just a simple "Log In". ChatGPT suggested "Welcome, please sign in."